geode-core/src/test/java/org/apache/geode/internal/admin/remote/DistributionLocatorIdJUnitTest.java

```diff
@@ -46,4 +48,25 @@ public void testEquals() throws UnknownHostException {
 
   }
 
+  @Test
+  public void testEquals_and_DetailCompare() throws UnknownHostException {
+    DistributionLocatorId dLI1 = new DistributionLocatorId(40404, "127.0.0.1", null);
```

Comment:
Let's change these variable names to something more description than an abbreviation:
```
DistributionLocatorId distributionLocatorId1 = new DistributionLocatorId(40404, "127.0.0.1", null);
```